### PR TITLE
Additional needhelp lines

### PR DESCRIPTION
**Pull request by LynxSolstice**
# About the pull request

Adds 9 new *medic lines. Each one either mentions Corpsman or Doc. Makes the TF2 easteregg lines more rare.

# Explain why it's good for the game

Extra voicelines, immersion (people will be calling for a Corpsman now, rather than a medic.), plus more variety in lines heard.

# Testing Photographs and Procedure

Compiles, runs, and works fine.

# Changelog

:cl:LynxSolstice
spellcheck: Changed the visible mentions of "Medic" to "Corpsman"
soundadd: Added 8 new lines for the *medic emote.
/:cl:

### CODE REVIEW

File: sound/voice/Im_hit_get_doc_up_here.ogg
Comment (harryob) [inline sound/voice/Im_hit_get_doc_up_here.ogg:1]: lowercase

File: sound/voice/I_need_a_corpsman_1.ogg
Comment (harryob) [inline sound/voice/I_need_a_corpsman_1.ogg:1]: lowercase

Comment (harryob) [inline sound/voice/I_need_a_corpsman_1.ogg:1]: these files are all still upper case btw

**Review by harryob — CHANGES_REQUESTED** [review]

File: sound/voice/I_need_a_corpsman_1.ogg
Comment (LynxSolstice) [reply sound/voice/I_need_a_corpsman_1.ogg:1]: Renamed em in VSC but I guess I gotta do it in the explorer, I'll push it in a min or 2 once I get it done.

Comment (LynxSolstice) [reply sound/voice/I_need_a_corpsman_1.ogg:1]: They don't seem to be uppercase in the file explorer OR VSC, they're just uppercase in github for some reason? 
image
image

**Pull request merged**